gtk4-macros/src/composite_template_derive.rs

```diff
@@ -54,7 +79,9 @@ pub fn impl_composite_template(input: &syn::DeriveInput) -> TokenStream {
 
     quote! {
         impl #crate_ident::subclass::widget::CompositeTemplate for #name {
-            fn bind_template_children(klass: &mut Self::Class) {
+            fn template_init(klass: &mut Self::Class) {
```

Comment:
Maybe rename this to set_template because that's what it does? this way we avoid the confusion with init_template

Reply:
Wouldn't that conflict with the underlying set_template? Also I did not like a set with no argument...

Reply:
good suggestion!